test/extended/jobs/jobs.go

```diff
@@ -9,7 +9,7 @@ import (
 	exeutil "github.com/openshift/origin/test/extended/util"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/kubernetes/pkg/apis/batch"
+	batchv1 "k8s.io/kubernetes/pkg/apis/batch/v1"
```

Comment:
Yes, we've explicitly called that out in 3.5 [upgrade doc](https://docs.openshift.com/container-platform/3.5/install_config/upgrading/manual_upgrades.html#additional-instructions-per-release) and that was added to [ansible](https://github.com/openshift/openshift-ansible/pull/3367).